/zenoh_pcd_gen:
  ros__parameters:
    interval_ms: 5
    payload_size: 1048576
    pub_topic: transfer_topic
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false
